runtime/lib.rs

```diff
@@ -76,11 +76,6 @@ pub static UNSTABLE_GRANULAR_FLAGS: &[(
     "Enable unstable file system APIs",
     4,
   ),
-  (
-    ops::http::UNSTABLE_FEATURE_NAME,
-    "Enable unstable HTTP APIs",
-    5,
-  ),
```

Comment:
This will not work. We still want this flag to exist, but not show in the help output. You'll need to add another field here (a boolean) that marks if this flag is supposed to show in the help output.